lms/lib/comment_client/user.py

```diff
@@ -167,9 +167,11 @@ def get_user_social_stats(user_id, course_id, end_date=None, thread_type=None):
         params.update({'end_date': end_date.isoformat()})
     if thread_type:
         params.update({'thread_type': thread_type})
+    if thread_ids:
+        params.update({'thread_ids': ",".join(thread_ids)})
 
     response = perform_request(
-        'get',
+        'post',
```

Comment:
@mtyaka discussion_id is a lengthy thing: about 48 characters or something, and GET is [de-facto limited to whatever the implementation limits it to (usually somewhere about 2000 characters)](http://stackoverflow.com/a/417184/882918). With this change it sends all cohorted discussion ids, so if get was used, course with more than 40 cohorted discussions might fail the export. It is unlikely to happen in production, but to stay on the safe side it was switched to post